presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java

```diff
@@ -69,6 +69,11 @@
 
 public class LogicalPlanner
 {
+    public enum Stage
```

Comment:
It is for tests ; /

Currently when plan is validated and find correlation (non rewritten Apply) will raise an exception, but in tests I want to get the plan before validation to check if correlation was resolved correctly.